### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by alexiscyber14**
Use semantic HTML tags.
Apply best practices in HTML code.
Use CSS selectors correctly.
Use CSS box model.
Use Flexbox to place elements in the page.
Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
Use GitHub Pages to deploy web pages.
Apply JavaScript best practices and language style guides in code.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.
Use objects to store and access data.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,22 +5,102 @@
...
+<h4 style="color:white;" class="bottom-button"><a href="">Book Your 2023 place today</a></h4>
+</div>
+</div>
+
+<div id="content2" class="content2">
+ <div class="liner">
+    <h3 class="main-program" >Featured Guides</h3>
+    <p class="line-one"></p>
+    </div>
+    <div class="wrapper-2">
```
Comment (frank1738) [inline index.html:83]: - [ ] Kindly note that the general background color of your body should be `#d3d3d3` and not white. 

image

### Checkout the original design

image

File: README.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+<p>see license here <a href="/LICENSE.md">license</a></p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (frank1738) [inline README.md:219]: - [ ] You have provided two links to your `MIT` license. Kindly  remove the first link as it is broken

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @alexiscyber14 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @alexiscyber14, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+<!-- FAQ (optional) -->
+
+- **[Is this code user friendly??]**
+
+  - [Yes, this code is user friendly and easy to understand]
+
+- **[Is teh link to clone the project free??]**
+
+  - [Yes it is. Follow the link to access this project<a href="https://github.com/alexiscyber14/vihiking-capstone">https://github.com/alexiscyber14/vihiking-capstone</a>]
```
Comment (brainconnect93) [inline README.md:213]: - [ ] [OPTIONAL but highly recommended] I would kindly advise you to acknowledge the owner of the real design used for this project in the `Acknowledgment` section of your README file. The Behance Design by Cindy Shin and add the link to the design in your readme file.

   - N.B: The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @alexiscyber14, 👋 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

- [ ] [Optional] I would strongly suggest you fix the merging conflict in your project as this will prevent you from merging your feature branch to the main branch if not fixed.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**